lib/commons/dom/is-visible.js

```diff
@@ -60,7 +76,9 @@ dom.isVisible = function(el, screenReader, recursed) {
 	if (
 		style.getPropertyValue('display') === 'none' ||
 		['STYLE', 'SCRIPT', 'NOSCRIPT', 'TEMPLATE'].includes(nodeName) ||
-		(!screenReader && isClipped(style.getPropertyValue('clip'))) ||
+		(!screenReader &&
+			(isClipped(style.getPropertyValue('clip')) ||
+				isClipped(style.getPropertyValue('clip-path')))) ||
```

Comment:
Your `isClipped` method takes two different inputs, based on which it does different things? I'm not sure that's a great idea. Either have two separate functions, or pass the style in and have `isClipped` check for both cases.